python/lsst/meas/algorithms/convertReferenceCatalog.py

```diff
@@ -262,6 +421,18 @@ def assertAllOrNone(*names):
             raise ValueError(
                 '"epoch_name" must be specified if "pm_ra/dec_name" or "parallax_name" are specified')
 
+        # Need all the error field names set if we are including covariances.
+        if self.full_position_information:
+            # Since coordinate_covariance is True, this will only pass for the
+            # "None" case
+            assertAllOrNone("full_position_information",
```

Comment:
I don't understand the comment above: won't this only pass for the "All" case, since all those fields must be set if `full_position_information` is True?

Reply:
Yeah, should be "all"...somehow it got reversed between my brain and fingers.